### DISCUSSION TITLE
유니티 빌드잇 연동

### DISCUSSION THREAD

**Discussion by TaeHanYoo** [naverz/zepeto-studio-kor#314] [4. General]
빌드잇에서 제작한 월드를 유니티 상에 임포트하고
유니티로 작업한 뒤 다시 빌드잇으로 업로드하는 것이 가능한가요?

빌드잇에서 구축한 월드를 유니티로 불러와 revit으로 제작한 bim model을 임포트하고 다시 빌드잇의 상호작용객체를 사용하고 싶습니다.

**Comment by hyeonjin-jo** [comment]
안녕하세요.

현재 빌드잇에서 제작한 월드를 유니티에 로드 후 다시 빌드잇쪽으로 업로드하는 것은 제공하지 않고 있습니다.

월드 제작 시에 빌드잇으로 만든 맵을 활용하는 것은 가능합니다.

https://studio.zepeto.me/guides/loading-buildit-maps
